server/handlers.go

```diff
@@ -98,10 +98,15 @@ func (svr *Server) handleGetOPGenericCommitment(w http.ResponseWriter, r *http.R
 	return svr.handleGetShared(r.Context(), w, commitment, commitmentMeta)
 }
 
-func (svr *Server) handleGetShared(ctx context.Context, w http.ResponseWriter, comm []byte, meta commitments.CommitmentMeta) error {
+func (svr *Server) handleGetShared(
+	ctx context.Context,
+	w http.ResponseWriter,
+	comm []byte,
+	meta commitments.CommitmentMeta,
+) error {
```

Comment:
did the linter config change or something?

Reply:
Seems like there's a lot of formatting changes bloating this PR :'(

Reply:
My IDE is configured to automatically reformat lines that grow beyond a reasonable limit, it's not part of the linter.

Personally I think that too-long-lines are substantial impediment to readability, which is why I have tended to commit such automatic changes. I can stop doing so, though, if others don't agree that long lines are a problem

Reply:
I'm fine with that, I just think it should be part of the linter config, otherwise someone else might come in and then refactor to the old style (if they have a longer max line length) and the CI will just let this back-and-forth go through

Reply:
+100 to having this as a linter enforced std

Reply:
>  I just think it should be part of the linter config,

Definitely agree. It's a big step to add it to the linter unfortunately, since it involves refactoring the whole repo for line length... In the future I'll be more cautious about my formatting preferences blowing up the size of a PR (particularly ones that are already large)